PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+- Refactor (refactoring code, removing useless files)
+- Unit tests (no changes in the code)
+- Bump-up dependent library (no changes in the code)
+- Behavioral tests (no changes in the code)
+- Integration tests (Kafka, Logstash etc.)
+- Benchmarks (no changes in the code)
+- Documentation update
+
+## Testing steps
+
+Please describe how the change was tested locally. If, for some reason, the testing was not done or not done fully, please describe what are the testing steps.
+
+## Checklist
+* [ ] `make before_commit` passes
+* [ ] updated documentation wherever necessary
+* [ ] added or modified tests if necessary
+* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change
```

Comment:
Is this necessary for this case?

Reply:
yup, the `engine` produces messages to be consumed by Aggregator, and another one by vuln. service